pennylane/estimator/templates/select.py

```diff
@@ -43,9 +43,8 @@ class SelectTHC(ResourceOperator):
     Args:
         thc_ham (:class:`~pennylane.estimator.compact_hamiltonian.THCHamiltonian`): A tensor hypercontracted
             Hamiltonian on which the select operator is being applied.
-        rotation_precision (int | None): The number of bits used to represent the precision for loading
-            the rotation angles for basis rotation. If :code:`None` is provided, the default value from the
-            :class:`~.pennylane.estimator.resource_config.ResourceConfig` is used.
+        rotation_precision (int): The number of bits used to represent the precision for loading
+            the rotation angles for basis rotation. The default value is set to ``15`` bits.
```

Comment:
Side comment, The `Resources:` section of the docstring for all of these Resource Operators could use more work. We should address this in one of the Qubitize PRs. As we market this feature more, people will be reading the docstrings and looking for clear answers and guidance for how the resources come about.